### PR TITLE
com_tags // avoid deprecation message is $element->core_body is empty

### PR DESCRIPTION
**Pull request by svenbluege**
Deprecation messages are shown for some tagged elements: 
image

Pull Request for Issue # .

### Summary of Changes
In com_tags, a tagged element's text-variable is transformed into null if the value is empty. 
That causes deprecation messages in plugins like loadmodules and emailcloak. They usually don't need to check for null values in $article->text. 

### Testing Instructions

1. configure PHP to show deprecation messages
2. create a new contact using com_contact: just enter a name and tag it with "testtag"
3. open the frontend /index.php?option=com_tags&view=tags
4. click on the new tag 'testtag'.
5. see the deprecation messages.

### Actual result BEFORE applying this Pull Request

Following the test instructions, deprecation messages are shown

### Expected result AFTER applying this Pull Request

Following the test instructions, deprecation messages are now shown

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: components/com_tags/src/View/Tag/HtmlView.php [hunk 1]
```diff
@@ -175,7 +175,7 @@ public function display($tpl = null)
                 $itemElement->event = new \stdClass();
 
                 // For some plugins.
-                !empty($itemElement->core_body) ? $itemElement->text = $itemElement->core_body : $itemElement->text = null;
+                !empty($itemElement->core_body) ? $itemElement->text = $itemElement->core_body : $itemElement->text = '';
```
Comment (Quy) [inline components/com_tags/src/View/Tag/HtmlView.php]:
```suggestion
                $itemElement->text = !empty($itemElement->core_body) ? $itemElement->core_body : '';
```

Comment (svenbluege) [reply components/com_tags/src/View/Tag/HtmlView.php]: Looks better, thanks. I added the change.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 803c9b71ca03ac24340d4ffe66f4797c4351c316<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39640">issues.joomla.org/tracker/joomla-cms/39640</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 803c9b71ca03ac24340d4ffe66f4797c4351c316<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39640">issues.joomla.org/tracker/joomla-cms/39640</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39640">issues.joomla.org/tracker/joomla-cms/39640</a>.</sub>

**Comment by fancyFranci** [comment]
Nice! Thank you!

**Comment by svenbluege** [comment]
@fancyFranci  That issue reappeared in 4.3-dev. It looks like that change did not make it through in the merge. I added a PR for 4.3 here: https://github.com/joomla/joomla-cms/pull/40048

**Pull request merged**